tests/slow/src/test/scala/tests/feature/DefinitionCrossLspSuite.scala

```diff
@@ -23,13 +23,15 @@ class DefinitionCrossLspSuite
     }
   }
 
-  if (super.isValidScalaVersionForEnv(BuildInfo.scala213)) {
-    test("2.13") {
-      basicDefinitionTest(BuildInfo.scala213)
+  if (super.isValidScalaVersionForEnv(BuildInfo.scala212)) {
+    test("2.12") {
+      basicDefinitionTest(BuildInfo.scala212)
     }
   }
 
-  test("underscore") {
+  // TODO MIGRATE I can't reproduce this locally. In CI this fails every time,
```

Comment:
Could you create a follow up issue for this?

Reply:
Yup, done.